### ISSUE TITLE
Cannot find module `../lang/undefined`?

### ISSUE THREAD

**Issue by JesseLeite** [AngryKiller/iTunes-Discord#4]
Getting this when I run `npm start` 🤔 

```
> itunes-discord@0.6.1 start /Users/jesseleite/Code/itunes-discord
> electron .

App threw an error during load
Error: Cannot find module '../lang/undefined'
    at Module._resolveFilename (module.js:543:15)
    at Function.Module._resolveFilename (/Users/jesseleite/Code/itunes-discord/node_modules/electron/dist/Electron.app/Contents/Resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.Module._load (module.js:473:25)
    at Module.require (module.js:586:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/jesseleite/Code/itunes-discord/utils/lang.js:9:15)
    at Object.<anonymous> (/Users/jesseleite/Code/itunes-discord/utils/lang.js:16:3)
    at Module._compile (module.js:642:30)
    at Object.Module._extensions..js (module.js:653:10)
    at Module.load (module.js:561:32)
```

**Comment by AngryKiller** [comment]
Can you check if the config file is correctly generated? It should be located at "/Users/yourname/Library/Application Support/iTunes-Discord" on macOS and "%AppData%/Roaming/iTunes-Discord" on Windows

**Comment by AngryKiller** [comment]
Which version of macOS are you using?

**Comment by JesseLeite** [comment]
10.14 Mojave.

**Comment by AngryKiller** [comment]
I'm still on High Sierra, so maybe electron-store is having issues with macOS Mojave. You can try creating a settings.json file in the folder where the config should be located and manually enter the config:
```json
{
"lang": "english",
"launch-at-login": false
}
```

**Comment by JesseLeite** [comment]
Hmm, didn't seem to work 🤔

**Comment by AngryKiller** [comment]
Same error?

**Comment by JesseLeite** [comment]
Yessir.  Btw, if it helps I can screenshare on discord, my ID is jesseleite#8296.

**Comment by JesseLeite** [comment]
Yessir. Btw, if it helps I can screenshare on discord, my ID is jesseleite#8296.

**Comment by AngryKiller** [comment]
fixed in commit a058005.
